src/provisioner/provisioner.py

```diff
@@ -85,6 +85,10 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
         except ConfError as e:
             Log.error(f'Unable to load {solution_config_url} url, Error:{e}')
 
+        # Secrets path from config file
+        if cortx_conf.get('cortx>common>storage>local'):
+            CortxProvisioner.secrets_path =cortx_conf.get('cortx>common>storage>local')+'/solution/secret'
```

Comment:
save relative path '/solution/secret' in a variable and then use it to form absolute path

Reply:
Yes, please use constant define for relative path: `/solution/secret`